apps/docs/package.json

```diff
@@ -3,6 +3,8 @@
   "scripts": {
     "dev": "next dev",
     "build": "next build",
+    "onegobuild": "next build && next-sitemap",
```

Comment:
might be better to rename it build:sitemap

Reply:
Sure.

Should I remove `postbuild` script?

Reply:
nope if I think it could be keep